client-participation/js/strings/sk.js

```diff
@@ -30,65 +30,65 @@ s.info = "Info";
 s.submitComment = "Zdieľať";
 s.tipStarred = "Označené ako dôležité.";
-s.participantHelpWelcomeText = "Vitajte v novom druhu online diskusie - <em>hlasujte</em> o návrhoch a pridajte svoje vlastné. Len tak sa priblížime ku konsenzu";
-s.participantHelpGroupsText = "Účastníci, ktorí hlasovali podobne <span style='font-weight: 700;'>sú zoskupení.</span> Kliknite na skupinu pre zobrazenie názorov, ktoré zdieľajú. <a style='font-weight: 700; cursor: pointer; text-decoration: underline' id='helpTextGroupsExpand'>...ďalšie</a>";
+s.participantHelpWelcomeText = "Vitajte v novom druhu online diskusie - <em>hlasujte</em> o návrhoch a zdieľajte vaše názory a skúsenosti. Len tak sa priblížime sa ku konsenzu";
+s.participantHelpGroupsText = "Diskutujúci, ktorí hlasovali podobne <span style='font-weight: 700;'>sú zoskupení v názorových skupinách.</span> Kliknite na jednotlivú skupinu pre zobrazenie názorov, ktoré zastávajú. <a style='font-weight: 700; cursor: pointer; text-decoration: underline' id='helpTextGroupsExpand'>...ďalšie</a>";
 s.participantHelpGroupsNotYetText = "Vizualizácia sa zobrazí keď 7 účastníkov začalo hlasovať";
-s.helpWhatAreGroupsDetail = '<p> Pravdepodobne ste už videli "odporúčané produkty" na Amazone, alebo "odporúčané filmy" na Netflixe. Všetky tieto internetové služby využívajú štatistiku na zoskupenie uživateľov, ktorí kupujú, či sledujú podobné veci. Tieto služby im následne odporúčajú produkty, ktoré si ľudia z ich skupiny už kúpili, či pozreli. </p> <p> Keď účastník konverzácie hlasuje o jednotlivých komentároch, tak algoritmus ho zoskupí s účastníkmi, ktorí hlasovali podobne. Grafické vyobrazenie týchto skupín môžte vidieť nižšie. Každá skupina je zložená z ľudí, kotrí majú podobný názor na problematiku. Práve to nám umožňuje viesť konštruktívnu diskusiu a zistiť unikátne poznatky. Neváhajte, kliknite na ľubovolnú skupinu nižšie a zistite čo nás rozdeľuje a čo spája. </p>';
-s.socialConnectPrompt = "Môžte si pripojiť svoj účet, aby ste vo vizualizácii videli vašich priateľov a ľudí, ktorých sledujete.";
+s.helpWhatAreGroupsDetail = '<p> Pravdepodobne ste už videli "odporúčané produkty" na Amazone, alebo "odporúčané filmy" na Netflixe. Všetky tieto internetové služby využívajú štatistiku na zoskupenie uživateľov, ktorí kupujú, či sledujú podobné veci. Tieto služby im následne odporúčajú produkty, ktoré si ľudia z ich skupiny už kúpili či pozreli.</p> <p> Keď účastník konverzácie hlasuje o jednotlivých komentároch, tak algoritmus ho zoskupí s účastníkmi, ktorí hlasovali podobne. Grafické vyobrazenie týchto skupín môžte vidieť nižšie. Každá skupina je zložená z ľudí, kotrí majú podobný názor na vec. Práve to nám umožňuje viesť konštruktívnu diskusiu a prehľadne zistiť čo si veľké skupiny ľudí myslia. Neváhajte, kliknite na ľubovolnú skupinu nižšie a zistite čo nás rozdeľuje a čo spája.</p>';
+s.socialConnectPrompt = "Optionally connect to see friends and people you follow in the visualization.";
```

Comment:
Did you intend to revert this to English?

Reply:
If you accept this reversion suggestion, then the tests will automatically re-run (since it will be a new code commit), and then hopefully we'll see a pass :)

```suggestion
s.socialConnectPrompt = "Môžte si pripojiť svoj účet, aby ste vo vizualizácii videli vašich priateľov a ľudí, ktorých sledujete.";
```

Reply:
Good catch. Thanks!